lib/entry/FlavorDownloader.js

```diff
@@ -509,6 +523,32 @@ var FlavorDownloader = (function() {
 
     }
 
+    function useSavedChunks(){
+        if (this.recordingStartTime)
+        {
+            for (let mp4Files of this.savedMp4Files)
+            {
+                let mp4FilesToUse = [];
+                for (let currMp4File of mp4Files)
```

Comment:
_.filter(mp4Files, currMp4file=> currMp4File.startTime + currMp4File.video.duration > this.recordingStartTime)